lib/editor/reducers/timetable.js

```diff
@@ -135,6 +136,10 @@ const timetable = (state: TimetableState = defaultState, action: Action): Timeta
       return update(state, {
         hideDepartureTimes: {$set: !state.hideDepartureTimes}
       })
+    case 'TOGGLE_USE_SECONDS':
+      return update(state, {
+        useSecondsInOffset: {$set: !state.useSecondsInOffset}
```

Comment:
When the useSeconds is disabled, it needs to set the seconds to `0`. Otherwise there is a phantom value that gets added and you can't remove it!

Reply:
Great catch thank you. Updated in https://github.com/ibi-group/datatools-ui/pull/984/commits/765e63123f7ad1f4e028f681e96929f82bdc9393